### PR TITLE
Removes dependency on implied CDN for image hosting

### PR DESCRIPTION
**Pull request by markmcd**
PTAL @skarEE @brendankenny

### CODE REVIEW

**Comment by markmcd** [comment]
#344

File: markerclusterer/examples/speed_test_example.html [hunk 1]
```diff
@@ -83,22 +83,14 @@
     <script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?sensor=false"></script>
     <script src="../src/data.json" type="text/javascript"></script>
     <script type="text/javascript">
-      var script = '<script type="text/javascript" src="../src/markerclusterer';
-      if (document.location.search.indexOf('compiled') !== -1) {
-        script += '_compiled';
-      }
-      script += '.js"><' + '/script>';
+      var script = '<script type="text/javascript" src="../src/markerclusterer.js"><' + '/script>';
```
Comment (samthor) [inline markerclusterer/examples/speed_test_example.html]: couldn't this just be inline now that there's no code controlling the src?

Comment (markmcd) [inline markerclusterer/examples/speed_test_example.html]: Yep - done.

**Comment by samthor** [comment]
LGTM

**Pull request merged**